src/game/server/ddracechat.cpp

```diff
@@ -3246,3 +3246,13 @@ void CGameContext::ConRainbowNameVIP(IConsole::IResult *pResult, void *pUserData
 
 	pChr->RainbowName(!pChr->GetPlayer()->m_RainbowName, pResult->m_ClientID);
 }
+
+void CGameContext::ConShrug(IConsole::IResult *pResult, void *pUserData)
+{
+    CGameContext *pSelf = (CGameContext *)pUserData;
+    CCharacter *pChr = pSelf->GetPlayerChar(pResult->m_ClientID);
+    if(!pChr)
```

Comment:
Not required to check for the character. I'd just check whether `m_apPlayers[pResult->m_ClientID]` exists, that's enough.